query/templates/search/v710/contentFilters.tmpl

```diff
@@ -1,14 +1,20 @@
-{{ if or .URIPrefix .TopicWildcard}}
 ,
 "filter" : [
-{{if or .URIPrefix .TopicWildcard}}
-  { "bool": { "should": [
-
-  {{ template "contentFilterOnURIPrefix.tmpl" . }}
-  {{ template "contentFilterOnTopicWildcard.tmpl" . }}
-
-    ]}
-  }
- {{end}}
+{
```

Comment:
Could we tidy the file up to make it easier to read, like so:

```suggestion
  {
    "bool": {
      "must": [
        {
          "bool": {
            "should": [
              {{ template "contentTypeFilter.tmpl". }}
            ]
          }
        }
        {{ if or.URIPrefix.TopicWildcard.Topic }}
        , {
          "bool": {
            "should": [
              {{ template "topicFilters.tmpl". }} 
              {{ template "contentFilterOnURIPrefix.tmpl". }}
              {{ template "contentFilterOnTopicWildcard.tmpl". }}
            ]
          }
        }
        {{ end }}
      ]
    }
  }
]
```

Reply:
Fixed in update 1.